Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:wheezy
+
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y python-pip python-virtualenv virtualenvwrapper python-dev libjpeg-dev libxml2-dev libxslt1-dev zlib1g-dev build-essential gettext
```

Comment:
Note to self: move that to a `make debiandeps` or something like this, so we don't need to update in the doc AND here next time we add a dependency.